eng/ci/official-release.yml

```diff
@@ -55,14 +55,13 @@ extends:
     stages:
     - stage: Prepare
       jobs:
-        - job: Prepare Release
+        - job: Prepare
 
           variables:
             drop_path: $(Pipeline.Workspace)/drop
 
           templateContext:
-            type: releaseJob
-            isProduction: true
```

Comment:
isProduction true is not needed anymore?

Reply:
It is not needed for `validationJob` type.